Assets/HoloToolkit-Examples/SharingWithUNET/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,135 @@
+﻿using UnityEngine;
```

Comment:
Same thing with this class. Or wrap it in a namespace so there won't be conflicts.